sdk-extensions/jaeger-remote-sampler/src/main/java/io/opentelemetry/sdk/extension/trace/jaeger/sampler/JaegerRemoteSampler.java

```diff
@@ -45,14 +46,16 @@
   private volatile Sampler sampler;
 
   JaegerRemoteSampler(
-      String serviceName,
+      @Nullable String serviceName,
       ManagedChannel channel,
       int pollingIntervalMs,
       Sampler initialSampler,
       boolean closeChannel) {
     this.channel = channel;
     this.closeChannel = closeChannel;
-    this.serviceName = serviceName;
+    // TODO(anuraaga): This probably needs to be replaced with using the Resource, but the spec
```

Comment:
snark

Reply:
why not? I don't think anyone objects to samplers being constructed with the Resource.

Reply:
I object to it for reasons I've mentioned several times I think :P I've given up on the train of thought since the UX issues don't seem to resonate with anyone it seems

Reply:
Anyways snarked the snark